environment/src/main/java/com/sequenceiq/environment/environment/validation/securitygroup/AwsEnvironmentSecurityGroupValidator.java

```diff
@@ -31,7 +32,7 @@ public void validate(EnvironmentDto environmentDto, ValidationResult.ValidationR
             if (onlyOneSecurityGroupIdDefined(securityAccessDto)) {
                 resultBuilder.error(securityGroupIdsMustBePresent());
             } else if (isSecurityGroupIdDefined(securityAccessDto)) {
-                if (!Strings.isNullOrEmpty(environmentDto.getNetwork().getNetworkCidr())) {
+                if (RegistrationType.CREATE_NEW.equals(environmentDto.getNetwork().getRegistrationType())) {
```

Comment:
is it possible that the getNetwork returns with null, because in that case, NPE would occur

Reply:
the network can not be null because we have converter for every platform